tests/test_imageset.py

```diff
@@ -17,6 +19,29 @@
 from dxtbx.model.experiment_list import ExperimentListFactory
 
 
+def test_single_file_indices(dials_regression):
+    from dxtbx.model.beam import BeamFactory
```

Comment:
please float imports
```suggestion
from dxtbx.model.beam import BeamFactory
from dxtbx.model.experiment_list import ExperimentListFactory


def test_single_file_indices(dials_regression):
```

Reply:
Thanks, @Anthchirp. I committed all of your changes.